best-practices.html

```diff
@@ -70,6 +71,15 @@ <h1>Best practices and recommended tools</h1>
         <main>
             <section id="all">
                 <h2>For all GitHub users <a href="#all">&#x1F517;</a></h2>
+                <section id="2fa">
+                    <h3>Enable 2FA <a href="#2fa">&#x1F517;</a></h3>
+                    <p>Multi-factor authentication, or two-factor authentication (<abbr title="Two-factor authentication">2FA</abbr>)
+                        is <a href="https://help.github.com/articles/about-two-factor-authentication/">a simple and efficient way to make our online identities
+                        and assets more robust against attacks</a>.
```

Comment:
I am not sure users would understand the benefit of 2FA for them when reading :
> to make our online identities and assets more robust against attacks

Here is a suggested update for this paragraph (taking text from the github about 2FA page and adding "to protect your online identity")

```html
<p><a href="https://help.github.com/articles/about-two-factor-authentication/">Two-factor authentication, or 2FA<a>, is an extra layer of security used when logging into websites or apps to protect your online identity. With 2FA, you have to log in with your username and password and provide another form of authentication that only you know or have access to.</p>
```